users.js

```diff
@@ -1293,13 +1465,17 @@ class User {
 			this.chatQueueTimeout = null;
 		}
 	}
+	/**
+	 * @return {undefined}
```

Comment:
This should be unnecessary?

Reply:
I think it has to have this, since it's recursive so TypeScript gets confused about the return type.

Reply:
Oh, I see.